test.sh

```diff
@@ -60,11 +60,19 @@ for version in "${meteor_versions[@]}"; do
 	if [[ "${version}" == 1.6* ]] || [[ "${version}" == 1.7* ]] || [[ "${version}" == 1.8* ]]; then
 		node_version='8.17.0'
 
-	# Versions 1.9 through 2.2.x need Node 12.22.3
-	elif [[ "${version}" == 1.9* ]] || [[ "${version}" == 1.10* ]] || [[ "${version}" == 1.11* ]] || [[ "${version}" == 1.12* ]] || [[ "${version}" == 2.0* ]] || [[ "${version}" == 2.1* ]] || [[ "${version}" == 2.2* ]]; then
-		node_version='12.22.3'
-
-	# Versions >= 2.3 need Node 14.17.3
+	# Versions 1.9 through 2.2 need Node 12.22.1
+	elif [[ "${version}" == 1.9* ]] || [[ "${version}" == 1.10* ]] || [[ "${version}" == 1.11* ]] || [[ "${version}" == 1.12* ]] || [[ "${version}" == 2.0* ]] || [[ "${version}" == 2.1* ]] || [[ "${version}" == 2.2 ]]; then
+		node_version='12.22.1'
+		
+	# Versions 2.2.1 need Node 12.22.2
+	elif [[ "${version}" == 2.2.1 ]]; then
+		node_version='12.22.2'
+		
+	# Versions 2.3 need Node 14.17.1
+	elif [[ "${version}" == 2.3 ]]; then
```

Comment:
You’ll need to distinguish between 2.3.0 and 2.3.1 below. Is 2.3 reported as `2.3` or as `2.3.0`? As in, does this script accurately return 14.17.1 for 2.3.0 and 14.17.3 for 2.3.1?

Reply:
That's a good question. I think that's correct, but just to be sure I'll double check by adding extract information on the CI logging

Reply:
If that's of interest, I was thinking to use something like this: 
https://github.com/chneau/meteor-base/blob/test_versions/versions.sh 
where the Node version and the Meteor version is split using:
```bash
meteor_version="${versions%___*}"
node_version="${versions#*___}"
```
This will need to be the same in `continuous-integration-workflow.yml`

Reply:
It looks to be good:
```md
PASS for geoffreybooth/meteor-base:2.3 with node:14.17.1-alpine
PASS for geoffreybooth/meteor-base:2.3.1 with node:14.17.3-alpine
```